app/models/event.rb

```diff
@@ -35,8 +35,6 @@ class Event < ActiveRecord::Base
   validates :program, presence: true
   validates :max_attendees, numericality: { only_integer: true, greater_than_or_equal_to: 1, allow_nil: true }
 
-  validate :max_attendees_no_more_than_room_size
```

Comment:
> Increasing the room.size is not realistic, rooms have a maximum size (consider fire regulations, or amount of workstations needed for a workshop to take place).
Then we shouldn't allow that an event with more attendees is scheduled in that room.